eng/build.yml

```diff
@@ -1,53 +1,53 @@
 parameters:
+  # Agent OS identifier and used as job name
   agentOs: ''
-  queue: {}
+
+  # Agent pool
+  pool: {}
+
+  # Additional variables
   variables: {}
 
-phases:
+  # Build strategy - matrix
+  strategy: {}
+
+  # Job timeout
+  timeoutInMinutes: 180
+
+jobs:
```

Comment:
You should move this to use the `jobs/jobs.yml` file or `job/job.yml` file.  https://github.com/dotnet/arcade/blob/master/Documentation/AzureDevOps/PhaseToJobSchemaChange.md#--what-is-the-engcommontemplatesjobjobyml-template

Reply:
Done.